releasenotes/notes/4872-Deprecate-Schedule-input-arg-schedules.yaml

```diff
@@ -0,0 +1,6 @@
+---
+deprecations:
+  - |
+    Argument `schedule` for :class:`qiskit.pulse.Schedule` has been
+    deprecated and will be removed in a later release.
+    You can still use `Schedule()` or `Schedule(name=)`.
```

Comment:
Similar to the deprecation warnings. `Constructing :class:`qiskit.pulse.Schedule`s with ...